VEDA_OS01/models.py

```diff
@@ -255,68 +426,48 @@ class Video (models.Model):
     video_trans_start = models.DateTimeField('Process Start', null=True, blank=True)
     video_trans_end = models.DateTimeField('Process Complete', null=True, blank=True)
 
-    """
-    TODO: STATUS REMODEL:
-    Change to
-    'Ingest',
-    'Queued',
-    'In Progress',
-    'Corrupt',
-    'Complete',
-    'Error',
-    'Duplicate',
-    'Review',
-    'Reject'
-
-    Possile:
-        'Invalid' (for ingest detected)
-        'Retry'
-        'Delivery' (for celery states?)
-
-    """
-
-    SI = 'Ingest'
-    TQ = 'Transcode Queue'
-    AT = 'Active Transcode'
-    TR = 'Transcode Retry'
-    TC = 'Transcode Complete'
-    DU = 'Deliverable Upload'
-    FC = 'File Complete'
-    CF = 'Corrupt File'
-    RH = 'Review Hold'
-    RR = 'Review Reject'
-    RP = 'Final Publish'
-    YD = 'Youtube Duplicate'
-    TRANS_STATUS_OPTIONS = (
-        (SI, "System Ingest"),
-        (TQ, "Transcode Queue"),
-        (AT, "Active Transcode"),
-        (TR, "Transcode Retry"),
-        (TC, "Transcode Complete"),
-        (DU, "Deliverable Upload"),
-        (FC, "File Complete"),
-        ('Transcode Error', "Transcode Error"),
-        (CF, "Corrupt File on Ingest"),
-        (RH, "Review Hold"),
-        (RR, "Review Rejected"),
-        (RP, "Review to Final Publish"),
-        (YD, "Youtube Duplicate"),
-        ('Queue', "In Encode Queue"),
-        ('Progress', "In Progress"),
-        ('Complete', "Complete")
-
-    )
     video_trans_status = models.CharField(
         'Transcode Status',
         max_length=100,
-        choices=TRANS_STATUS_OPTIONS,
-        default=SI
+        choices=VideoStatus.CHOICES,
+        default=VideoStatus.SI
     )
 
     video_glacierid = models.CharField('Glacier Archive ID String', max_length=200, null=True, blank=True)
     abvid_serial = models.CharField('VEDA Upload Process Serial', max_length=20, null=True, blank=True)
     stat_queuetime = models.FloatField('Video Avg. Queuetime (sec)', default=0)
 
+    # 3rd Party Transcription
+    process_transcription = models.BooleanField('Process transcripts from Cielo24/3PlayMedia', default=False)
+    provider = models.CharField(
+        'Transcription provider',
+        max_length=20,
+        choices=TranscriptProvider.CHOICES,
+        null=True,
+        blank=True,
+    )
+    three_play_turnaround = models.CharField(
+        '3PlayMedia Turnaround',
+        max_length=20,
+        choices=ThreePlayTurnaround.CHOICES,
+        null=True,
+        blank=True,
+    )
+    cielo24_turnaround = models.CharField(
+        'Cielo24 Turnaround', max_length=20,
```

Comment:
hope you've verified the max_length for all these fields, i would prefer to have a considerable cushion in max_length for these fields.